### ISSUE TITLE
If I press "run" the build process will build the binary, attempt to run, but then hang

### ISSUE THREAD

**Issue by dangmoody** [slynch8/10x#687]
If I build my EXE and then want to run it via the "run" command then I get the following output from the build tab:

```
----- Build config platform -----
Executing build command:
build.bat
===== Success : 0 errors, 0 warnings =====
Executing build command:
test.exe
CreateProcess error: The system cannot find the file specified.
```

I've attached a repro project if you want it (but it does use the same convention as in my previous issue of calling clang and assuming it's in your PATH etc.) but this time I am also uploading the prebuilt .exe so you can just hit run and (hopefully) repro; hope that helps!

For the repro project I have tried setting my `exe path` setting in my workspace to the following to try and get 10x to find the correct path, but it doesn't do anything for me:

- empty
- `.` (with and without trailing forward/backward slashes)
- The absolute path of the exe (with and without trailing forward/backward slashes)

Am I just fundamentally using 10x wrong in this instance?

I've seen other _similar_ looking issues in the issue tracker so apologies if this gets treated as a duplicate.

[10x_bug_building_and_running.zip](https://github.com/slynch8/10x/files/7961482/10x_bug_building_and_running.zip)

**Comment by slynch8** [comment]
looks like I'm not passing in the working directory. I'll get that fixed.

**Comment by slynch8** [comment]
thanks for the test project, that was very helpful. This is fixed in the latest build (0.30.38)
